fuzzing/README.md

```diff
@@ -374,6 +374,19 @@ https://www.freetype.org/freetype2/docs/reference/ft2-bitmap_handling.html
 | 23.1.6 | `FT_GlyphSlot_Own_Bitmap` |                    | Glyph Slot          |
 | 23.1.7 | `FT_Bitmap_Done`          | :heavy_check_mark: | Library <br> Bitmap |
 
+### COLRv1 Methods
+
+Not yet released, see documentation in [freetype.h](https://gitlab.freedesktop.org/freetype/freetype/-/blob/master/include/freetype/freetype.h#L4137)
+
+|    | Function                   | Fuzzed             |
+| -- | -------------------------- | :----------------: |
+|    | `FT_Get_Color_Glyph_Paint` | :heavy_check_mark: |
+|    | `FT_Get_Paint_Layers`      | :heavy_check_mark: |
+|    | `FT_Get_Paint_Layers`      | :heavy_check_mark: |
+|    | `FT_Get_Colorline_Stops`   | :heavy_check_mark: |
+|    | `FT_Get_Paint`             | :heavy_check_mark: |
```

Comment:
@cherusker: Right now, all of the COLR v2 stuff is part of the 'Glyph Layer Management' section, having the following order (including structures and the like).
```
FT_LayerIterator
FT_Get_Color_Glyph_Layer
FT_PaintFormat
FT_ColorStopIterator
FT_ColorIndex
FT_ColorStop
FT_PaintExtend
FT_ColorLine
FT_Affine23
FT_Composite_Mode
FT_OpaquePaint
FT_PaintColrLayers
FT_PaintSolid
FT_PaintLinearGradient
FT_PaintRadialGradient
FT_PaintSweepGradient
FT_PaintGlyph
FT_PaintColrGlyph
FT_PaintTransformed
FT_PaintTranslate
FT_PaintRotate
FT_PaintSkew
FT_PaintComposite
FT_COLR_Paint
FT_Color_Root_Transform
FT_Get_Color_Glyph_Paint
FT_Get_Paint_Layers
FT_Get_Colorline_Stops
FT_Get_Paint
```
I'm open to suggestions to improve that, if you feel this might be useful.

Reply:
Resources column added.